### ISSUE TITLE
In terra button group, cannot focus on buttons in Safari

### ISSUE THREAD

**Issue by gerard-isdell** [cerner/terra-core#3619]
# Bug Report

## Description
When I open this page in Safari (Version 15.4 )

https://engineering.cerner.com/terra-ui/raw/components/cerner-terra-core-docs/button-group/about

and I click on the "up" button in the Examples section, I cannot use the tab key to tab to the next button in the group (down) and have it take focus.  

## Steps to Reproduce

1. Open this url in safari https://engineering.cerner.com/terra-ui/raw/components/cerner-terra-core-docs/button-group/about
2. Click on Up button in example
3. Press tab

## Expected Behavior
After pressing the tab key I would expect the down button to have focus

## Environment

* Terra Button Group
* Safari Desktop 15.4:
* Node/npm Version: [e.g. Node 8/npm 5] n/a
* Webpack Version: n/a
* Operating System and version (desktop or mobile): Mac OS X 12.3.1

**Comment by bjankord** [comment]
1. Open the Safari browser on your Mac.
2. Select Safari > Preferences > Advanced.
3. To make it easier to navigate links and form elements, check the option 'Press Tab to highlight each item on a webpage.'
<img width="785" alt="Screen Shot 2022-05-12 at 7 14 07 AM" src="https://user-images.githubusercontent.com/633148/168072416-279510e7-3db8-4cde-9344-ee3ffc7e8b44.png">

**Comment by gerard-isdell** [comment]
All working for me now in Safari - Thanks Brett :-)

**Comment by benbcai** [comment]
Thanks @bjankord for helping out with this!

@gerard-isdell I'm going to close this issue if things are working for you in Safari.
